projects/laji/src/app/+project-form/form/form.component.ts

```diff
@@ -107,8 +107,12 @@ export class FormComponent implements OnInit {
 
   tryRedirectToSubForm(form: Form.SchemaForm, routeParams: Params): Observable<boolean> {
```

Comment:
[just a comment]: This method seems to be called from ngOnInit which is ran rather late in the component initialization that can case small flicker from time to time. I'm wondering if we could move most of this logic to canActivate guard and make the redirect there as needed?

Reply:
Good idea - I added a separate task for this https://www.pivotaltracker.com/story/show/176574492